### ISSUE TITLE
Installing Handbrake for Video_Transcoding

### ISSUE THREAD

**Issue by mumblequatch** [donmelton/video_transcoding#336]
I've successfully run Video_Transcoding in the past many times, but suddenly when I try to do so, I get this error:

/usr/local/bin/transcode-video: HandBrakeCLI not available

I have HandBrake installed via the GUI Mac OS app, but I don't know if that's sufficient. If a Linux install would potentially solve the issue, I cannot for the life of me figure out how to do this via Mac Terminal (FlatPak??). 

If that's not the solution, I would love any suggestions as to what might be instead. My noobiness is extreme enough that I will need actual step-by-step instructions, including command line entries — I don't really understand any shorthand!

**Comment by hughker** [comment]
TL;DR: you need to install it separately follow the instructions [here](https://github.com/donmelton/video_transcoding#requirements).

---

hi @mumblequatch — you’ll need to install it via Homebrew by following the instructions under [Requirements](https://github.com/donmelton/video_transcoding#requirements) which I’ve also pasted below for quick reference. The tool isn’t able to use the GUI and thus requires the CLI version to be installed as well. 

However, you can still keep the GUI if you plan to use it for your self converting other files (I personally keep both installed as sometimes it just needs to be a quick+dirty convert and I don’t need to leverage @donmelton tool).

Homebrew is pretty user friendly, but please reach out if you’re having issues working with it and I’m sure we can get you squared away!

---

### INSTRUCTIONS FROM README FOR QUICK REFERENCE

> On macOS, HandBrakeCLI and all its other dependencies can be easily installed via [Homebrew](http://brew.sh/), an add-on package manager:
> ```
> brew install handbrake
> brew install ffmpeg
> brew install mkvtoolnix
> brew install mp4v2
> ```
> 
> The optional crop previewing package can also be installed via Homebrew:
> ```
> brew install mpv
> ````

**Comment by mumblequatch** [comment]
Thank you for this! I tried the instructions you shared and now I get this error:

zsh: command not found: brew

I do appear to have a bunch of Homebrew stuff on my hard drive and yesterday I somehow was able to use it to install the  other helper apps for video_transcode. Hmmm...

/Users/thomaswoodrow/homebrew

/Users/thomaswoodrow/homebrew/Library/Homebrew

/Users/thomaswoodrow/homebrew/Library/Taps

**Comment by hughker** [comment]
@mumblequatch — enter `which brew` into the terminal and report back what it says for the output on the next line… should be something similar to `/usr/local/bin/brew`

if not, then i would do the Homebrew install again to see if it can fix any errors it had during installation:

```
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

also, keep an eye out for any errors that are towards the end of the Homebrew install.

**Comment by donmelton** [comment]
@hughker Thanks for your very clear and very helpful comments!

@mumblequatch You can use the `--filter` option to apply any of the HandBrake filters. While the `deinterlace` filter is applied automatically for some inputs, you may need to apply it yourself. Try adding `--filter deinterlace` to your command line. If that doesn't improve your output, you can try the `decomb` or `detelecine` filters instead.

I hope that helps.
